src/main/resources/views/CommercialVehicles/MSVA30.hbs

```diff
@@ -38,7 +38,6 @@
 </div>
 
 
-<div class="separate-page">
```

Comment:
It's likely CB2-12602 will go into develop prior to this one, and therefore should be resolved by the rebase. But to be sure, in the event that is not the case can you bump the template versions to `Version 1.2 July 2024` please.